build.gradle

```diff
@@ -21,7 +21,7 @@ plugins {
   id 'io.spring.dependency-management' version '1.0.11.RELEASE'
   id 'org.springframework.boot' version '2.5.2'
   id 'com.github.ben-manes.versions' version '0.38.0'
-  id 'hmcts.ccd.sdk' version '0.23.2'
+  id 'hmcts.ccd.sdk' version '0.23.4'
```

Comment:
this should already be in master... I think

Reply:
Merged changes from master it has got latest sdk version - 0.23.4